CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.40]
+
+### Added
+* Added a quick link to our [MintPy tutorial notebook](https://nbviewer.jupyter.org/github/ASFHyP3/hyp3-docs/blob/main/docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb)
```

Comment:
I thought it was the Using HyP3 Python SDK notebook that was added.

Reply:
But maybe the quick link is something different, and I should add a changelog entry for adding the menu item for the Using HyP3 Python SDK notebook.... Because I'm not quite clear on this, I'll leave it to @jhkennedy.

Reply:
@hjkristenson that's refering to this redirect I added:
https://github.com/ASFHyP3/hyp3-docs/blob/develop/mkdocs.yml#L96

That way we can give out a like like `https://hyp3-docs.asf.alaska.edu/tutorials/mintpy` and it'll automagically redirect to the notebook.

I can add a changelog entry for the sdk though

Reply:
PR here: ASFHyP3/hyp3-docs#235